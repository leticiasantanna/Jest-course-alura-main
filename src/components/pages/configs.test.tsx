import { render } from "@testing-library/react";
import { RecoilRoot } from "recoil";
import Config from "./Configs";

const mockNavigate = jest.fn();
jest.mock("react-router-dom", () => {
  return {
    useNavigate: () => mockNavigate,
  };
});
describe("page of configs", () => {
  it("must be render conrrectly", () => {
    const { container } = render(
      <RecoilRoot>
        <Config />
      </RecoilRoot>
    );
    expect(container).toMatchSnapshot();
  });
});
