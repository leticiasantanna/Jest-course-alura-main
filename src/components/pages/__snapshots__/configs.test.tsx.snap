// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page of configs must be render conrrectly 1`] = `
<div>
  <form
    class="sc-bcXHqe"
  >
    <input
      placeholder="Adicione os nomes dos participantes"
      type="text"
      value=""
    />
    <button
      disabled=""
    >
      Adicionar
    </button>
    
  </form>
  <ul />
  <footer>
    <button
      disabled=""
    >
      Iniciar brincadeira
    </button>
  </footer>
</div>
`;
